content/blog/getting-started-with-private-cloud-business-edition-apis.md

```diff
@@ -0,0 +1,365 @@
+        },
+        {
+            "createdAt": "2024-02-20T02:56:40Z",
+            "customerId": "00000000000000000000000000000000",
+            "eula": "https://update.nimblestorage.com/catalog/download/eula-rel-juno-1100.html",
+            "generation": 1447550,
+            "hypervisor": null,
+            "hypervisorManager": null,
+            "id": "6b692b44-ce0d-44a2-8a19-4d9cbfd9330f",
+            "name": "1.77",
+            "releaseDate": "2024-02-11",
+            "resourceUri": "/private-cloud-business/v1beta1/system-software-catalogs/6b692b44-ce0d-44a2-8a19-4d9cbfd9330f",
+            "serverFirmware": null,
+            "storageConnectionManager": null,
+            "storageSoftware": null,
+            "systemsWithUpdatePath": null,
+            "type": "private-cloud-business/system-software-catalog",
+            "updatedAt": "2024-05-03T13:32:55Z",
+            "version": "1.77"
+        },
+        {
+            "createdAt": "2024-03-11T01:57:07Z",
+            "customerId": "00000000000000000000000000000000",
+            "eula": "https://update.nimblestorage.com/catalog/download/eula-rel-pebble-400.html",
+            "generation": 644305,
+            "hypervisor": null,
+            "hypervisorManager": null,
+            "id": "f66777dd-e380-4319-9a32-a7ee24ff4cd4",
+            "name": "7.61.35.18.37",
+            "releaseDate": "2024-03-05",
+            "resourceUri": "/private-cloud-business/v1beta1/system-software-catalogs/f66777dd-e380-4319-9a32-a7ee24ff4cd4",
+            "serverFirmware": null,
+            "storageConnectionManager": null,
+            "storageSoftware": null,
+            "systemsWithUpdatePath": null,
+            "type": "private-cloud-business/system-software-catalog",
+            "updatedAt": "2024-05-03T13:31:52Z",
+            "version": "7.61.35.18.37"
+        },
+        {
+            "createdAt": "2024-03-11T01:57:05Z",
+            "customerId": "00000000000000000000000000000000",
+            "eula": "https://update.nimblestorage.com/catalog/download/eula-rel-pebble-400.html",
+            "generation": 990969,
+            "hypervisor": null,
+            "hypervisorManager": null,
+            "id": "3f3a18be-00f8-4478-9a40-b0b316d47771",
+            "name": "2.99",
+            "releaseDate": "2024-03-05",
+            "resourceUri": "/private-cloud-business/v1beta1/system-software-catalogs/3f3a18be-00f8-4478-9a40-b0b316d47771",
+            "serverFirmware": null,
+            "storageConnectionManager": null,
+            "storageSoftware": null,
+            "systemsWithUpdatePath": null,
+            "type": "private-cloud-business/system-software-catalog",
+            "updatedAt": "2024-05-03T13:32:55Z",
+            "version": "2.99"
+        }
+    ],
+    "offset": 0,
+    "total": 11
+}
+```
+
+In the next blog in this series we will have a look at a more secure implementation of the client_secret, using a key chain manager to read the client_secret.
```

Comment:
Suggestion: 

In the next blog post of this series you will have a look at a more secure...

or

In the next blog post of this series, I will explore a more secure implementation of...

Also, a suggestion to add something like that:

If you have any questions on the HPE GreenLake for Private Cloud Business Edition APIs, or if you are interested sharing your feedback and use cases on this set of APIs, please join the HPE Developer Slack [Workspace](https://developer.hpe.com/slack-signup) and start a discussion in our [#hpe-greenlake-data-services](https://hpedev.slack.com/archives/C02D6H623JP) Slack channel.